app/scripts/storage/index.js

```diff
@@ -12,4 +12,11 @@ const ThirdPartyStorage = {
     onedrive: require('./storage-onedrive')
 };
 
-module.exports = _.extend({}, BuiltInStorage, ThirdPartyStorage);
+let storages;
+if (Launcher.name === 'cordova') {
```

Comment:
Please don't check Launcher this way. Instead, create a property `canUseStorages` in Launcher and check it, but not here.
Remove this code to StorageBase, there's `enabled` property in it.